.vimrc

```diff
@@ -31,6 +31,12 @@ Plug 'tpope/vim-surround'
 
 Plug 'w0rp/ale'
 
+" Fulltext search
+" Requirements:
+" brew install the_silver_searcher
+Plug '/usr/local/opt/fzf'
+Plug 'junegunn/fzf.vim'
```

Comment:
We need update it to `Plug 'junegunn/fzf', { 'dir': '~/.fzf', 'do': './install --all' }`
or delete and add to requirements `brew install fzf`.
btw it's cool plugin, thanks @olegokunevych

Reply:
My bad, everything is ok with this line.